docs/skill_json.md

```diff
@@ -43,12 +43,12 @@ On the JSON file:
 * `skill_id` Required unique skill identifier, this is defined in `setup.py` and by convention is usually `repo.author` (lower-cased)
 * `source` Optional git url to download the source code, a skill can be installed from source with `pip install git+{source}`
```

Comment:
Looking at this again, there are [other supported VCS](https://pip.pypa.io/en/stable/topics/vcs-support/). Rather than guessing this is a git source, perhaps this field should be informational and `pip_spec` required to specify a valid spec.